### PR TITLE
OEL-1300: Integrate the extended project content type.

### PR DESCRIPTION
**Pull request by Maxfire**

### CODE REVIEW

**Review by drishu — COMMENTED** [review]
OK for now, two considerations:
1. the fields might need to be revisited after merge of the whitelabel PR
2. for future I will create a ticket to move these content type related config into a suibmodule, but not now because it should be done right and include news and event

**Review by drishu — CHANGES_REQUESTED** [review]
One thing I realized we should add is alias pattern like here https://github.com/openeuropa/oe_showcase/blob/1.x/modules/oe_showcase_page/config/install/pathauto.pattern.page_url_alias_pattern.yml

**Review by drishu — CHANGES_REQUESTED** [review]
1) Please double check that anonymous users can see project nodes and skos.
2) Please add a test called **ProjectTest.php** in  tests/src/ExistingSite that:

- asserts users without permission dont have access to the project creation page
- set up assets programatically (images or whatever u need to have ready)
- with a user that has permissions create a project **through the UI**
- save, go to the node url and assert the values are present

File: tests/src/ExistingSite/ProjectTest.php [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    $page->fillField('Objective', 'Text Objective');
+    $page->fillField('Impacts', 'Text Impacts');
+    $page->fillField('Achievements and milestones', 'Text Achievements and milestones');
+
+    // Participants.
+    $page->pressButton('Add new participant');
+    $page->fillField('Name', 'Developer participant name');
+    $page->fillField('Acronym', 'Developer participant acronym');
+    $page->fillField('Country', 'BE');
+    $page->fillField('Contribution to the budget', '19.9');
```
Comment (donquixote) [inline tests/src/ExistingSite/ProjectTest.php:100]: The selectors are ambiguous.

Comment (donquixote) [reply tests/src/ExistingSite/ProjectTest.php:100]: Also, are we not forgetting fields?
Lead contributors?

File: tests/src/ExistingSite/ProjectTest.php [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    $assert_session->pageTextContains('Objective');
+    $assert_session->pageTextContains('Text Objective');
+    $assert_session->pageTextContains('Impacts');
+    $assert_session->pageTextContains('Text Impacts');
+    $assert_session->pageTextContains('Achievements and milestones');
+    $assert_session->pageTextContains('Text Achievements and milestones');
+
+    // Assert participants.
+    $assert_session->pageTextContains('Participants');
+    $assert_session->pageTextContains('Name');
```
Comment (donquixote) [inline tests/src/ExistingSite/ProjectTest.php]: We should use some kind of selector.
Not sure what's the best solution here.

Comment (donquixote) [reply tests/src/ExistingSite/ProjectTest.php]: Also, "Name" alone is not very useful to test.
We want to verify that "Name" appears left of "Developer participant name", and all that within the participants section.

**Review by donquixote — CHANGES_REQUESTED** [review]

**Review by drishu — DISMISSED** [review]
enough for now

**Review by abel-santos-corral — DISMISSED** [review]

**Review by drishu — APPROVED** [review]
ok for now

**Review by donquixote — APPROVED** [review]

**Pull request merged**